config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Webplatform::Application.routes.draw do
 
+  get 'organiser' =>'organiser#index'
```

Comment:
please use `resources :organizers, only: [:index]`. We'll implement all CRUD actions for an organizer so it makes sense to use resource here.